Message/Handler/WarmupCacheHandler.php

```diff
@@ -14,14 +14,13 @@
 use Liip\ImagineBundle\Imagine\Filter\FilterManager;
 use Liip\ImagineBundle\Message\WarmupCache;
 use Liip\ImagineBundle\Service\FilterService;
-use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
 
 /**
  * Listen to WarmupCache messages and prepare the cache for those images.
  *
  * @experimental
  */
-class WarmupCacheHandler implements MessageHandlerInterface
+class WarmupCacheHandler
```

Comment:
The interface is removed in favor of the `AsMessageHandler` attribute for autowiring, but since the service isn't autowired, the attribute isn't really necessary here.